guides/common/modules/con_installing-a-project-server.adoc

```diff
@@ -0,0 +1,34 @@
+[id="installing-a-{project-context}-server_{context}"]
+= Installing a {ProjectServer}
+
+Installing an instance of {ProjectServer} on a dedicated server is the first step to a working {Project} infrastructure.
+ifdef::satellite[]
+You can install a {ProjectServer} in a connected or disconnected setup:
+
+* Connected deployment is suitable for networked environments where your {ProjectServer} is connected to the Red{nbsp}Hat CDN.
+* Disconnected deployment is suitable for high-security environments where direct Internet access is restricted or prohibited.
+
+A disconnected {ProjectServer} is isolated from Red{nbsp}Hat CDN but you can still provision systems with the latest security updates, errata, packages, and other content.
+You can use the following methods to import content to a disconnected {ProjectServer}:
+
+Content ISO::
+In this setup, you download ISO images with content from the Red{nbsp}Hat Customer Portal and extract them to {ProjectServer} or a local web server.
+The content on {ProjectServer} is then synchronized locally.
++
+This allows for complete network isolation of {ProjectServer}, however, the release frequency of content ISO images is around six weeks and not all product content is included.
+
+Disconnected {Project} with {ISS}::
+In this setup, you install a connected {ProjectServer} and export content from it to populate a disconnected {Project} using a storage device.
++
+This allows for exporting both Red{nbsp}Hat provided and custom content at the frequency you choose, but requires deploying an additional server with a separate subscription.
+endif::[]
+
+.Additional resources
+ifndef::satellite[]
+* For complete information on installing a {ProjectServer}, including prerequisites, predefined tuning profiles, see {InstallingServerDocURL}[_{InstallingServerDocTitle}_].
+endif::[]
+ifdef::satellite[]
+* For complete information on installing a {ProjectServer}, including prerequisites, predefined tuning profiles, see the following documents:
+** {InstallingServerDocURL}[_{InstallingServerDocTitle}_] for installing in a connected network
```

Comment:
suggestion: drop everything after the title; the title already mentions this. source: https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-titles.adoc#L48

same for line 33

Reply:
Done